src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -148,6 +148,7 @@ public String getQuestionTypeDisplayName() {
     public String getQuestionWithExistingResponseSubmissionFormHtml(
             boolean sessionIsOpen, int qnIdx, int responseIdx, String courseId,
             int totalNumRecipients,
+            boolean questionIsCompulsory,
```

Comment:
Not always consistently enforced in our code (we already have a violation of it 2 lines above this), but boolean variables/parameters should be prefixed with `is`. Let's try to not add more code violating our coding standards in.